stamp_generation/final_generated_stamp.html

```diff
@@ -136,71 +136,55 @@
 </head>
 <body>
     <amp-story
-        standalone title="This is title of website"
-        publisher="im.whatshot.in"
-        publisher-logo-src="https://www.google.com/s2/favicons?domain=im.whatshot.in"
+        standalone title="Despite Guaranteeing AAP's Landslide Victory, There Are Sadly No Women On Its Cabinet. Why?"
+        publisher="www.scoopwhoop.com"
+        publisher-logo-src="https://www.google.com/s2/favicons?domain=www.scoopwhoop.com"
         poster-portrait-src="abc.jpg">
         
             <amp-story-page id="page-1">
                 <amp-story-grid-layer template="fill" >
                     <amp-img class="bg-image"
-                         src="https://im.whatshot.in/img/2020/Mar/h-1584953735.jpg">
+                         src="https://s4.scoopwhoop.com/anj2/5e47ae1450758d46025b494d/17a659b6-07de-4b7c-8613-582286b455ab.jpg">
                     </amp-img>
                 </amp-story-grid-layer>
                 <amp-story-grid-layer template="fill" class="bg-solid" >
                 </amp-story-grid-layer>
                 <amp-story-grid-layer template="fill" class="drop-shadow media">
                     <div class="fixed-container">
                     <amp-img object-fit="contain" layout="fill"
-                         alt="Image Title"
-                         src="https://im.whatshot.in/img/2020/Mar/h-1584953735.jpg"
-                         width="100"
-                         height="100">
+                         alt="None"
+                         src="https://s4.scoopwhoop.com/anj2/5e47ae1450758d46025b494d/17a659b6-07de-4b7c-8613-582286b455ab.jpg"
+                         width="0"
+                         height="0">
                     </amp-img>
                     </div>
                 </amp-story-grid-layer>
                 
                 <amp-story-grid-layer template="vertical">
                     <amp-fit-text min-font-size="16" max-font-size="22" layout="responsive" class="page-title" >
-                        This is page1 of the generated stamp page!
+                        None
```

Comment:
My bad! I will update Jinja template so that **None** is not rendered in any attribute.

Reply:
I've to add functionality to render titles also into the stamp object - it's an issue rn - will address this in a PR.